portality/models/oaipmh.py

```diff
@@ -66,19 +66,21 @@ def list_sets(self):
         sets = [t.get("term") for t in result.get("facets", {}).get("sets", {}).get("terms", [])]
         return sets
 
-    def list_records(self, from_date=None, until_date=None, oai_set=None, list_size=None, start_number=None):
+    def list_records(self, from_date=None, until_date=None, oai_set=None, list_size=None, start_number=None, start_after=None):
```

Comment:
It looks like we can remove start_number from the method signature now

Reply:
The reason we need to keep the start number is so that the resumption token has the correct cursor and complete_list_size.

Reply:
ok, I see the point in keeping it across the board.  In this method, though, that argument is never used so isn't it redundant?  And the same as in the other methods in models/oaipmh.py.

Reply:
Oh yes and Now fixed.